models/api-models.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection')
+const fs = require("fs/promises");
+
+exports.showEndpoints = () => {
+    return fs.readFile('./endpoints.json', "utf-8").then(fileRead => {
```

Comment:
We already have the endpoints and Express can do this part for us, so it's probably better just to send the endpoints , or if you want to read the file, it can be done in the controller